tests/unit/AgmExtension.spec.ts

```diff
@@ -74,8 +74,6 @@ describe('AGM Extension view', () => {
   })
 
   afterEach(() => {
-    // restore feature flag
-    vi.restoreAllMocks()
```

Comment:
The comment is now wrong. As for the code, I moved it to where it was needed only.